app/components/organen/mandataris-table.hbs

```diff
@@ -78,12 +83,19 @@
           {{row.mandataris.bekleedt.bestuursfunctie.label}}
         </LinkTo></td>
       <td>
-        {{row.mandataris.status.label}}
+        <Mandaat::MandatarisStatusPill @mandataris={{row.mandataris}} />
```

Comment:
Shouldn't this have the `@detailView=false`? Otherwise that argument is never used/

Reply:
huh weird in your example it does show the beeindigd in the overview, for me it shows the niet beschikbaar, seems like it defaults to true for you and false for me if not mentioned, weird.

Reply:
this is a screenshot from an earlier version, your comments also seem to refer to an earlier version... i've pushed everything, did your pull come through correctly?

Reply:
detailView is falsy by default, and only true for the mandataris card

Reply:
didn't have the latest changes indeed